Appveyor is failing

Appveyor seems to have recently started failing on all builds.  Anyone have an idea what changed?  The failure comes right at the beginning while trying to install OpenCL:

```
Scanning dependencies of target OpenCL
[  4%] Building C object CMakeFiles/OpenCL.dir/icd.c.obj
System.ArgumentException: Illegal characters in path.
   at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
   at System.IO.Path.IsPathRooted(String path)
   at CClash.Compiler.ProcessArguments(String[] args) in c:\Users\inb\Desktop\cclash\cclash\CClash\Compiler.cs:line 554
```

See https://ci.appveyor.com/project/peastman/openmm/build/1.0.1347 for an example of a full build log.

There were a [bunch of commits to https://github.com/KhronosGroup/OpenCL-Headers on May 16](https://github.com/KhronosGroup/OpenCL-Headers) that moved everything into [newly named subdirectories](https://github.com/KhronosGroup/OpenCL-Headers). That seems consistent with failure to find headers:
```
c:\opencl\icd.h(44): fatal error C1083: Cannot open include file: 'CL/cl.h': No such file or directory
NMAKE : fatal error U1077: 'C:\projects\openmm\cclash-0.3.14\cl.exe' : return code '0x2'
Stop.
```
I think [this line](https://github.com/pandegroup/openmm/blob/master/appveyor.yml#L42) might need updating.

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -39,7 +39,7 @@ install:
   - ps: 7z x opencl-icd-1.2.11.0.tar > $null
   - ps: mv .\icd\* .
   - ps: mkdir inc/CL > $null
-  - ps: wget https://github.com/$opencl_github | select -ExpandProperty links | where {$_.href -like "*.h*"} | select -ExpandProperty outerText | foreach{ wget https://raw.githubusercontent.com/$opencl_github/master/$_ -OutFile inc/CL/$_ }
+  - ps: wget https://github.com/$opencl_github/tree/master/opencl20/CL | select -ExpandProperty links | where {$_.href -like "*.h*"} | select -ExpandProperty outerText | foreach{ wget https://raw.githubusercontent.com/$opencl_github/master/opencl20/CL/$_ -OutFile inc/CL/$_ }
   - ps: mkdir lib > $null
   - ps: cd lib
   - cmake -G "NMake Makefiles" ..

